src/components/SubscriptionTable/__tests__/SubscriptionTable.test.tsx

```diff
@@ -141,26 +155,28 @@ describe('Subscription Table', () => {
     ]);
 
     it('pages', () => {
-      const { container } = render(<Table subscriptions={get('data')} />);
-      expect(container).toMatchSnapshot();
+      render(<Table subscriptions={get('data')} />);
```

Comment:
I feel like this tests that the pager shows up, but not that the data pages. We should probably check that the length of data in the table is the correct page size if there is more data provided